hw/ip/i2c/doc/_index.md

```diff
@@ -336,10 +336,10 @@ In both cases, the `trans_complete` interrupt is asserted, in the beginning of a
 If an I2C target receives a START signal followed by an address and R/W = 1 (read), accepts a read transaction and its TX FIFO is empty, the interrupt `tx_empty` is asserted to inform firmware.
 The interrupt `tx_empty` is asserted also when a target keeps transmitting data on the bus and needs more data, but TX FIFO is empty.
 
-If a target receives a STOP or RESTART signal and there are extra bytes left in TX FIFO, the interrupt `tx_nonempty` is asserted and the FIFO is flushed.
+If a target receives a STOP or repeated START signal and there are extra bytes left in TX FIFO, the interrupt `tx_nonempty` is asserted and the FIFO is flushed.
 
-When a host receives enough data from a target, it usually signals the end of the transaction by sending a NACK followed by a STOP or a RESTART.
-In a case when a target receives an ACK and then a STOP/RESTART, the interrupt `tx_ack_stop` is asserted.
+When a host receives enough data from a target, it usually signals the end of the transaction by sending a NACK followed by a STOP or a repeated START.
+In a case when a target receives an ACK and then a STOP/START, the interrupt `tx_ack_stop` is asserted.
```

Comment:
Heh, not the greatest name for the interrupt, is it? Since stop and repeated start are very different. Something to fix on the hardware side, possibly.

Reply:
This interrupt isn't mentioned anywhere else here so I guess it's been renamed and this line didn't get fixed... I'll fix it and I've filed #15081 to track the question

Reply:
Thanks for filing the issue!